### DISCUSSION TITLE
Visibility of passwords at generation

### DISCUSSION THREAD

**Discussion by MaxWinsemius** [dani-garcia/vaultwarden#4463] [Q&A]
I recently upgraded my server from version 2.25.0 to 2024.1.2 (2.25.0 might correspond to 1.23.1 if I look at the commit hash), and I noticed that when generating new items my passwords are now shown. Not necessarily bad since they are still complex, but I can't find where I can change the setting that it's hidden by default.

This is what is currently shown:
![image](https://github.com/dani-garcia/vaultwarden/assets/12040226/2523cf94-fe91-4de9-9cec-4ee5d5e4bfbf)

This is what was previously shown:
![image](https://github.com/dani-garcia/vaultwarden/assets/12040226/93a2a09f-9190-4297-a5dc-301e53b0c0cd)

**Comment by Gerardv514** [comment]
This should be tested on Bitwarden to see if the issue exists there; if it does this needs to be reported upstream. If not then it’s a Vaultwarden bug. 

Looking at Bitwarden documentation it seems it should be hidden. I think I even remember some thread somewhere talking about this but can’t find it. Below is the documentation, and as per screenshot it’s hidden; unless they manually hid the password but that would be dumb. 

Update: This seems to work properly in the iOS client, but not web vault; which I believe is an upstream issue. 

https://bitwarden.com/help/managing-items/

**Reply by stefan0xC** [reply]
It's a bug that has been fixed by https://github.com/bitwarden/clients/pull/7666, so should be fixed once we update to the newest web-vault.
